### PR TITLE
Add the possibility to configure Confluent Stream catalog limit

### PR DESCRIPTION
**Pull request by ThomasCAI-mlv**

### CODE REVIEW

**Review by loicgreffier — COMMENTED** [review]
Hi @ThomasCAI-mlv, here is the review

File: src/main/java/com/michelin/ns4kafka/property/ConfluentCloudProperties.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    /**
+     * Stream Catalog properties.
+     */
+    @Getter
+    @Setter
+    @ConfigurationProperties("ns4kafka.confluent-cloud.stream-catalog")
+    public static class StreamCatalogProperties {
+        private int pageSize = 500;
+    }
+
```
[the remark this answers is not in the document]
Comment (ThomasCAI-mlv) [reply src/main/java/com/michelin/ns4kafka/property/ConfluentCloudProperties.java:25]: Not sure what do you mean by trimming. Is it putting the class out of ConfluentCloudProperties?

File: src/main/java/com/michelin/ns4kafka/property/ConfluentCloudProperties.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+@ConfigurationProperties("ns4kafka.confluent-cloud")
+public class ConfluentCloudProperties {
+    private StreamCatalogProperties streamCatalog;
+
+    /**
+     * Stream Catalog properties.
+     */
+    @Getter
+    @Setter
+    @ConfigurationProperties("ns4kafka.confluent-cloud.stream-catalog")
```
[the remark this answers is not in the document]
Comment (ThomasCAI-mlv) [reply src/main/java/com/michelin/ns4kafka/property/ConfluentCloudProperties.java:21]: yes

File: src/main/java/com/michelin/ns4kafka/property/ConfluentCloudProperties.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (ThomasCAI-mlv) [reply src/main/java/com/michelin/ns4kafka/property/ConfluentCloudProperties.java]: Maybe it is removing the line 25, I see

**Pull request merged**